rfcs/137-service-output-template.md

```diff
@@ -0,0 +1,35 @@
+- Author: Xuanwo <github@xuanwo.io>
+- Start Date: 2021-07-07
+- RFC PR: [beyondstorage/specs#137](https://github.com/beyondstorage/specs/issues/137)
+- Tracking Issue: [beyondstorage/go-storage#0](https://github.com/beyondstorage/go-storage/issues/0)
+
+# GSP-137: Service Output Template
+
+## Background
+
+[Increase code coverage](https://github.com/beyondstorage/go-storage/issues/620) plans to move `cmd/definitions/tests` into `tests` and support generating files into `generate_test.go` so we won't export test services.
+
+So `definitions` should have the ability to generate all files into a specified output filename template.
```

Comment:
Do we actually need this feature in `definitions`? If we just want to change `tests`, we can simply write some scripts in `Makefile`?

Reply:
> Do we actually need this feature in `definitions`? If we just want to change `tests`, we can simply write some scripts in `Makefile`?

You are right. The feature is used in `tests` services only. I will abort this GSP and use scripts instead.